arm-graphrbac/1.6/swagger/graphrbac.json

```diff
@@ -270,14 +270,14 @@
           "ApplicationKeyCredentials"
         ],
         "operationId": "Applications_ListKeyCredentials",
-        "description": "Get keyCredentials associated with the application by object Id. Reference: https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/entity-and-complex-type-reference#keycredential-type",
+        "description": "Get the keyCredentials associated with an application.",
```

Comment:
Yes, there are quite a few other opportunities for improvement with similar format. Could do for another pass, for sure.